projects/packages/jetpack-mu-wpcom/src/features/custom-css/custom-css/preprocessors/lessc.inc.php

```diff
@@ -1269,7 +1270,7 @@ protected function lib_contrast($args) {
 
 	protected function lib_luma($color) {
 	    $color = $this->coerceColor($color);
-	    return (0.2126 * $color[0] / 255) + (0.7152 * $color[1] / 255) + (0.0722 * $color[2] / 255);
+	    return (0.2126 * (float)$color[0] / 255) + (0.7152 * (float)$color[1] / 255) + (0.0722 * (float)$color[2] / 255);
```

Comment:
Looking at `coerceColor`, I don't think this is the correct fix here. Instead, it looks like the function is just wrong.

```suggestion
	    return (0.2126 * $color[1] / 255) + (0.7152 * $color[2] / 255) + (0.0722 * $color[3] / 255);
```

Reply:
Ah, great catch!